lib/Elastica/Response.php

```diff
@@ -78,14 +78,56 @@ public function __construct($responseString, $responseStatus = null)
      */
     public function getError()
     {
-        $message = '';
+        $error = $this->getFullError();
+
+        if (!$error) {
+            return '';
+        }
+
+        if (is_string($error)) {
+            return $error;
+        }
+
+        if (isset($error['root_cause'][0])) {
+            $error = $error['root_cause'][0];
```

Comment:
Thanks for looking into this - the code was copied from the master branch (maybe you want to look there too). I did not really get much into semantics, but I don't see why root cause would be an array of strings.